test/test_all/environment_test.cpp

```diff
@@ -60,23 +60,23 @@ void test_environment_sun_vector() {
   // Calculate and check for t=0.0 (comparing against MATLAB)
   gnc::env::sun_vector(0.0L, s);
   TEST_ASSERT_FLOAT_VEC_NEAR(1e-5, s, lin::Vector3f({
-     0.997341623364311f,
```

Comment:
Why did this change?

Reply:
Same reason as above. The sun vector model got updated in MATLAB at this point and not in C++.

Reply:
In this branch in MATLAB I'm getting:
```
>> config
>> env_sun_vector(0)

ans =

    0.9973
   -0.0668
   -0.0290
```

Reply:
This makes sense because PAN epoch is in march, near the vernal equinox.

Reply:
Fixed.